plugins/connection/docker.py

```diff
@@ -226,7 +226,7 @@ def _set_conn_data(self):
 
         # TODO: this is mostly for backwards compatibility, play_context is used as fallback for older versions
         # docker arguments
-        self._docker_args.clear()
+        del self._docker_args[:]
```

Comment:
Is this not supposed to work with 2.9?

Reply:
Do you mean Ansible 2.9? How's this related to Ansible 2.9?

Reply:
nvmd, this is a local array ... had confused with pc 'special' structuring of attirbutes